src/views/Hero/Hero.scss

```diff
@@ -0,0 +1,11 @@
+.HeroView {
+  height: 100%;
+}
+
+.HeroView__homeView {
```

Comment:
It looks like both the `HeroView` and `HeroView__homeView` class are applied to the same element. Can we use the `&` operator to eventually nest the `HeroView__homeView` class inside the `HeroView` class and make things a little bit cleaner?

Reply:
You're right! This is an [SCSS thing](https://sass-lang.com/documentation/style-rules/parent-selector). Good catch Tim! My css brain got to me there hehe